src/citrine/resources/file_link.py

```diff
@@ -0,0 +1,209 @@
+"""A collection of FileLink objects."""
+from uuid import UUID
+import os
+import mimetypes
+from boto3 import client as boto3_client
+from botocore.exceptions import ClientError
+
+from taurus.entity.file_link import FileLink as TaurusFileLink
+from citrine._serialization.properties import String
+from citrine._rest.collection import Collection
+from citrine._rest.resource import Resource
+from citrine._session import Session
+
+
+class _Uploader(object):
```

Comment:
You don't need `object` here

Reply:
I remembered learning once that it is good practice to always have objects extends `object`. Your comment made me look up why, and it has to do with reasons that have not been valid since Python 2.